src/shared/devnode-acl.c

```diff
@@ -123,14 +123,13 @@ int devnode_acl(const char *path,
                 }
         }
 
-        if (!changed)
-                return 0;
-
-        if (acl_calc_mask(&acl) < 0)
-                return -errno;
+        if (changed) {
```

Comment:
I do not see any merits of the second commit. If nothing is changed, then we can safely return the function earlier. This commit increase unnecessary indentation.